src/main/java/com/powsybl/openloadflow/network/impl/LfBranchImpl.java

```diff
@@ -165,7 +165,11 @@ public BranchResult createBranchResult(double preContingencyBranchP1, double pre
         var branchResult = new BranchResult(getId(), p1.eval() * PerUnit.SB, q1.eval() * PerUnit.SB, currentScale1 * i1.eval(),
                                             p2.eval() * PerUnit.SB, q2.eval() * PerUnit.SB, currentScale2 * i2.eval(), flowTransfer);
         if (createExtension) {
-            branchResult.addExtension(OlfBranchResult.class, new OlfBranchResult(piModel.getR1(), piModel.getContinuousR1()));
+            branchResult.addExtension(OlfBranchResult.class, new OlfBranchResult(piModel.getR1(), piModel.getContinuousR1(),
```

Comment:
You have three issues here:
- Bus can be null yes, but maybe you can find a way to hide all of this in `OlfBranchResult` constructor.
- v is still in per-unit quantities and we don't want that as output.
- angle is still in radians but output in degrees.